### ISSUE TITLE
Script stopped loading - "syntax error"

### ISSUE THREAD

**Issue by helifino** [WazeDev/WME-Place-Harmonizer#266]
As of 1.3.68, WMEPH (beta) throws an error and refuses to load.

Following is pasted from the Developer Console log:

Syntax error @ "WME Place Harmonizer Beta"!
##########################
JSHINT output:
require: (W041) Use '===' to compare with 'null' on line: 28 at character: 87
require: (W041) Use '!==' to compare with 'null' on line: 46 at character: 158
require: (W030) Expected an assignment or function call and instead saw an expression on line: 89 at character: 65
require: (W030) Expected an assignment or function call and instead saw an expression on line: 96 at character: 200
require: (W030) Expected an assignment or function call and instead saw an expression on line: 152 at character: 530
require: (W030) Expected an assignment or function call and instead saw an expression on line: 231 at character: 1137
require: (W030) Expected an assignment or function call and instead saw an expression on line: 241 at character: 195
require: (W030) Expected an assignment or function call and instead saw an expression on line: 255 at character: 712
require: (W030) Expected an assignment or function call and instead saw an expression on line: 341 at character: 1683
require: (W030) Expected an assignment or function call and instead saw an expression on line: 404 at character: 947
require: (W030) Expected an assignment or function call and instead saw an expression on line: 466 at character: 149
require: (W030) Expected an assignment or function call and instead saw an expression on line: 500 at character: 301
require: (W041) Use '===' to compare with 'null' on line: 528 at character: 754
require: (W041) Use '!==' to compare with 'null' on line: 623 at character: 600
require: (W041) Use '!==' to compare with 'null' on line: 756 at character: 2001
script: (W004) 'i' is already defined on line: 953 at character: 26
script: (W004) 'newName' is already defined on line: 2228 at character: 33
script: (W004) 'nameToCheck' is already defined on line: 4044 at character: 29
script: (W004) 'brand' is already defined on line: 4098 at character: 21
script: (W004) 're' is already defined on line: 4100 at character: 25
script: (W004) 'msgAdd' is already defined on line: 4186 at character: 37
script: (W004) 'i' is already defined on line: 4411 at character: 42
script: (W004) 're' is already defined on line: 4518 at character: 17
script: (W038) 'PNHMatchData' used out of scope on line: 4099 at character: 21
script: (W038) 'PNHMatchData' used out of scope on line: 4099 at character: 37
script: (W038) 'PNHMatchData' used out of scope on line: 4101 at character: 41
script: (W038) 'ph_speccase_ix' used out of scope on line: 4099 at character: 50
script: (W038) 'ph_speccase_ix' used out of scope on line: 4101 at character: 54
script: (W038) 'inferredAddress' used out of scope on line: 4173 at character: 58
script: (W038) 'inferredAddress' used out of scope on line: 4173 at character: 77
script: (W084) Expected a conditional expression and instead saw an assignment on line: 8077 at character: 36
script: (W084) Expected a conditional expression and instead saw an assignment on line: 8133 at character: 37
##########################

SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode
    at Function (native)
    at Object.L.E_c (<anonymous>:3:298)
    at ha (eval at <anonymous> (unknown source), <anonymous>:61:376)
    at Object.Ta.create (eval at <anonymous> (unknown source), <anonymous>:70:259)
    at f (eval at <anonymous> (unknown source), <anonymous>:15:237)(anonymous function) @

**Comment by mapomatic** [comment]
Thanks for the report.  I haven't been able to reproduce that error.  Are you seeing that in Chrome?

**Comment by helifino** [comment]
Yes. It's an older version though, as I am on an old computer that is no longer supported by the bleeding edge of either major browser.

**Comment by mapomatic** [comment]
Version 1.3.67 works for you?

**Comment by helifino** [comment]
The previous version worked for me, yes.
